exercises/verify.cpp

```diff
@@ -23,15 +27,11 @@ int main()
   cin >> number;
 
   // placeholder
-  int N[dim] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
+  int numbers[dim] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
 
-  for (int i=0; i<dim; i++) {
-    if (N[i] == number) {
-      cout << "The number " << number << " is present in the array" << endl;
-      return EXIT_SUCCESS;
-    }
-  }
+  if (check_number(numbers, dim, number))
+    cout << "The number " << number << " is present in the array" << endl;
+  else cout << "The number " << number << " is not present in the array" << endl;
 
```

Comment:
ciò che cambia qui è solo il "not", potresti usare la condizione ternaria per rendere tutto più compatto:
```C++
cout << "The number " << number << " is " << check_number(numbers, dim, number) ? "" : "not " << "present in the array" << endl;
```